Mathlib/Order/Filter/AtTopBot.lean

```diff
@@ -884,12 +884,10 @@ theorem map_neg_atTop : map (Neg.neg : β → β) atTop = atBot :=
   (OrderIso.neg β).map_atTop
 #align filter.map_neg_at_top Filter.map_neg_atTop
 
-@[simp]
 theorem comap_neg_atBot : comap (Neg.neg : β → β) atBot = atTop :=
```

Comment:
Should the statement be changed here to `-atBot = atTop`? Then it could still be `simp`.

Reply:
Oh nevermind, I guess that's the lemma in the other file.

This whole section looks like it should be generalized to `OrderedCommGroup`...